src/tutorial/11-Custom-Settings.md

```diff
@@ -32,12 +32,12 @@ Keys have one of three types. `SettingKey` and `TaskKey` are described in
 Some examples from [Keys](../sxr/sbt/Keys.scala.html):
 
 ```scala
-val scalaVersion = settingKey[String]("The version of Scala used for building.")
-val clean = taskKey[Unit]("Deletes files produced by the build, such as generated sources, compiled classes, and task caches.")
+val scalaVersion = settingKey[String]("scala-version", "The version of Scala used for building.")
```

Comment:
The previous code is still the *conceptual* constructor.  We would have it be exactly that if we weren't maintaining compatibility w/ 0.12 build scripts.